README.md

```diff
@@ -2,6 +2,7 @@
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Python](https://img.shields.io/badge/Language-Python-blue.svg)](https://python.org/)
+[![Build Status](https://github.com/CSCI-GA-2820-SP23-003/customers/actions/workflows/tdd.yml/badge.svg)](https://github.com/CSCI-GA-2820-SP23-003/customers/actions/workflows/tdd.yml)
```

Comment:
Not sure if it's a mandate, but the link should be till /actions as per the lab.
So, it should be:
```[![Build Status](https://github.com/CSCI-GA-2820-SP23-003/customers/actions/workflows/tdd.yml/badge.svg)](https://github.com/CSCI-GA-2820-SP23-003/customers/actions)```

Reply:
Yeah, I am assuming we are gonna have future badges (BDD testing for instance) so this would make it clearer.